utils/docker/Dockerfile.centos.8

```diff
@@ -33,27 +33,68 @@ gpgcheck=False\n" >> /etc/yum.repos.d/local-centos-group.repo;		\
 
 # Install basic tools to build DAOS
 
-RUN dnf clean all && dnf -y install epel-release dnf-plugins-core &&	\
-    dnf config-manager --set-enabled powertools &&			\
-    dnf -y upgrade &&							\
-    dnf -y install							\
-	boost-devel clang-analyzer cmake CUnit-devel doxygen file flex	\
-	gcc gcc-c++ git golang graphviz libaio-devel			\
-	libcmocka-devel libevent-devel libiscsi-devel libtool		\
-	libtool-ltdl-devel libuuid-devel libyaml-devel make nasm	\
-	numactl-devel openssl-devel pandoc patch glibc-langpack-en	\
-	readline-devel python3-scons sg3_utils yasm diffutils		\
-	pciutils valgrind-devel man fuse3-devel hwloc-devel patchelf	\
-	json-c-devel lz4-devel libipmctl-devel ndctl ipmctl		\
-	e2fsprogs							\
-	python2 python2-devel python36-devel python3-magic		\
-	python3-pygit2 python3-pygithub					\
-	python2-requests python3-requests				\
-	python2-Cython python3-pylint python3-pyxattr			\
-	python3-clustershell python3-paramiko				\
-	python3-numpy python3-jira Lmod					\
-	python3-tabulate python2-distro python3-distro			\
-	java-1.8.0-openjdk maven openmpi-devel libunwind-devel &&	\
+RUN dnf clean all && dnf -y install epel-release dnf-plugins-core && \
```

Comment:
Are there any changes to the packaging list here? This formatting is a welcome change but it would be much preferable to have one PR update the format, then simply insert and new packages in this PR so we can see both what's in this change and the history better.

The other thing I'd considered was to have this package list in a external file and copy it in, that way it would be easier to sort it and it wouldn't take up so much room in this file, although I doubt it would be possible to re-use the list in any of our other code.